src/aura/STG_CMP_System/STG_CMP_System.cmp

```diff
@@ -252,19 +252,19 @@
                 <div class="slds-col slds-size--1-of-2 slds-m-bottom--medium">
                     {!$Label.c.stgRefreshAdminAcctNameTitle}
                     <div class="slds-col slds-size--1-of-1">
-                    	<ui:outputText value="{!$Label.c.stgRefreshAdminAcctNameDesc}" class="slds-text-body--small" />
+                        <ui:outputText value="{!$Label.c.stgRefreshAdminAcctNameDesc}" class="slds-text-body--small" />
                     </div>
                 </div>
 
-              	<div class="slds-col slds-size--1-of-2">
-              		<lightning:button
+                  <div class="slds-col slds-size--1-of-2">
+                      <lightning:button
                                 disabled="{!!v.isView}"
-                        	    variant="brand"
-                    		    label="Refresh All Administrative Names"
-                        	    title="refreshButton"
-                        	    onclick="{! c.handleRefreshAdminAccount}"/>
+                                variant="brand"
+                                label="{!$Label.c.stgBtnRefreshAdminNames}"
+                                title="refreshButton"
+                                onclick="{! c.handleRefreshAdminAccount}"/>
```

Comment:
Not quite related to this item but maybe we can use the chance to get rid of the extra spaces here. There is another place too.

Reply:
I'll take a look, will have to wait until tomorrow though.